<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogoMediosFacebook</class>
 <widget class="QWidget" name="DialogoMediosFacebook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Páginas de Facebook</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layout_general">
    <item>
     <layout class="QFormLayout" name="layout_opciones">
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_etiqueta">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Etiqueta</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineedit_etiqueta">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>10</pointsize>
          <italic>true</italic>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_nombre_pagina">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nombre de página</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btn_agregar">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Agregar</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btn_limpiar">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Limpiar</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="btn_eliminar">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Eliminar</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineedit_nombre_pagina">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="layout_lista">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="lista_medios_facebook">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="btnbox_medios_facebook">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>360</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <action name="action_actualizar_y_cerrar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Actualizar y cerrar</string>
   </property>
   <property name="toolTip">
    <string>Actualizar y cerrar</string>
   </property>
  </action>
  <action name="action_resetear">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Resetear</string>
   </property>
   <property name="toolTip">
    <string>Resetear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_guardar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="toolTip">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="action_elimina">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
   <property name="toolTip">
    <string>Eliminar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_estado_btn_eliminar">
   <property name="text">
    <string>estado btn_eliminar</string>
   </property>
   <property name="toolTip">
    <string>estado btn eliminar</string>
   </property>
  </action>
  <action name="action_estado_btn_agregar">
   <property name="text">
    <string>estado btn_agregar</string>
   </property>
   <property name="toolTip">
    <string>estado btn_agregar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>lista_medios_facebook</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>action_estado_btn_eliminar</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>586</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
